src/compass/schemas/s1_cslc_geo.yaml

```diff
@@ -10,12 +10,14 @@ runconfig:
            safe_file_path: list(str(), min=1)
            # Required. List of orbit (EOF) files
            orbit_file_path: list(str(), min=1)
-           # Required. The unique burst ID to process
-           burst_id: str(required=True)
+           # Required. The unique burst IDs to process
+           burst_id: list(str(), min=1, required=False)
 
         dynamic_ancillary_file_group:
            # Digital Elevation Model.
            dem_file: str(required=False)
+           # burst database sqlite file
+           db_file: str(required=False)
```

Comment:
Also, since we decided to grab the parameters necessary to construct the geogrid from the burst database, I think that this portion of the schema (defaults) should be cleaned up:

```
geocoding:
   bottom_right:
       x: 
       y: 
       
   top_left:
       x: 
       y: 
     
```

However, we should leave to the user the tuning of the output posting in X/Y coordinates

Reply:
Do we want to keep these in case the user doesn't provide a database? Or are we making the database a requirement for all users?